src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -154,6 +151,12 @@ private async Task<IWebHost> BuildWebHost(ScriptApplicationHostOptions hostOptio
                 {
                     configBuilder.AddEnvironmentVariables();
                 })
+                .ConfigureLogging(loggingBuilder =>
+                {
+                    loggingBuilder.ClearProviders();
+                    loggingBuilder.AddDefaultWebJobsFilters();
+                    loggingBuilder.AddProvider(new ColoredConsoleLoggerProvider((cat, level) => level >= LogLevel.Information));
```

Comment:
This is now mirroring what we do in the host here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/Program.cs#L71-L76